website/docs/d/organization_members.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization_members"
+sidebar_current: "docs-datasource-tfe-organization-members"
+description: |-
+  Get information on an Organization members.
+---
+
+# Data Source: tfe_organization_members
+
+Use this data source to get information about members of an organization.
+
+## Example Usage
+
+```hcl
+resource "tfe_organization" "bar" {
+   name = "org-bar"
+   email = "user@hashicorp.com"
+}
+
+data "tfe_organization_members" "foo" {
+  organization = tfe_organization.bar.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `organization` - (Required) Name of the organization.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Name of the organization.
+* `members` - A list of active members of the organization.
+* `members_waiting` - A list of members with pending invite to organization.
+
+The `member` block contains:
+
+* `user_id` - The ID of the user.
+* `membership_id` - The ID of the membership.
```

Comment:
Elsewhere in the provider, we call this the `organization_membership_id` (`tfe_team_organization_member` resource) and colloquially "organization membership id" in other places (`tfe_organization_membership` resource and data source)

I think we should reuse this long name so we don't risk id confusion. WDYT?

Reply:
this sounds reasonable to me, I will update the name to match.